src/swarm/neo/connection/ConnectionBase.d

```diff
@@ -1034,6 +1048,23 @@ abstract class ConnectionBase: ISelectClient
 
     /***************************************************************************
 
+        Requests to send outgoing data buffered by the OS for the socket
+        immediately. By default outgoing data are buffered to the maximum
+        payload of a TCP frame using the Linux TCP_CORK socket option.
+
+        This method is intended to be called right after `getPayloadForSending`
+        has called the `send` delegate.
+
+    ***************************************************************************/
+
+    public void flush ( )
+    {
+        this.sender.flush();
```

Comment:
Ok. @david-eckardt-sociomantic could you extend the doc a bit to explain the reason for these two cases?